curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/check-for-mixed-grouping-of-characters.english.md

```diff
@@ -23,12 +23,14 @@ Then fix the code so that the regex that you have created is checked against <co
 
 ```yml
 tests:
-  - text: Your regex <code>myRegex</code> should return <code>true</code> for the string <code>"Hello"</code>
-    testString: assert(myRegex.test('Hello'), 'Your regex <code>myRegex</code> should return <code>true</code> for the string <code>"Hello"</code>');
-  - text: Your regex <code>myRegex</code> should return <code>true</code> for the string <code>"Halo"</code>
-    testString: assert(myRegex.test('Halo'), 'Your regex <code>myRegex</code> should return <code>true</code> for the string <code>"Halo"</code>');
-  - text: Your regex <code>myRegex</code> should return <code>false</code> for the string <code>"Hell"</code>
-    testString: assert(!myRegex.test('Hell'), 'Your regex <code>myRegex</code> should return <code>false</code> for the string <code>"Hell"</code>');
+  - text: Your regex <code>myRegex</code> should return <code>true</code> for the string <code>"Franklin Roosevelt"</code>
```

Comment:
Don't quote strings in code tags. We're programmers teaching programmers and one of the things to learn is the value of being precise, exact. With your instruction I'd try to match the quote marks as well, which is not what you're testing against.